libraries/CMakeLists.txt

```diff
@@ -46,4 +46,12 @@ foreach(module IN LISTS standard_modules )
     if(IS_DIRECTORY "${module}" AND EXISTS "${module}/CMakeLists.txt")
         add_subdirectory(${module})
     endif()
-endforeach()
\ No newline at end of file
+endforeach()
+
+# Add all CMakeLists.txts in the aws folder.
+file(GLOB aws_modules "${MODULES_DIR}/aws/*")
+foreach(module IN LISTS aws_modules )
+    if(IS_DIRECTORY "${module}" AND EXISTS "${module}/CMakeLists.txt")
+        add_subdirectory(${module})
+    endif()
+endforeach()
```

Comment:
You can use the same foreach loop instead of making another:
```c
# Add all modules.
file(GLOB standard_modules "${MODULES_DIR}/standard/*")
file(GLOB aws_modules "${MODULES_DIR}/aws/*")
foreach(module IN LISTS standard_modules aws_modules)
    if(IS_DIRECTORY "${module}" AND EXISTS "${module}/CMakeLists.txt")
        add_subdirectory(${module})
    endif()
endforeach()
```

Reply:
Thanks, will fix it